src/title/TitleStream.js

```diff
@@ -18,17 +18,20 @@ export default class TitleStream extends React.PureComponent {
 
   props: {
     subscriptions: [],
+    streams: [],
     narrow: () => {},
     color: string,
   };
 
   render() {
-    const { narrow, subscriptions, color } = this.props;
-    const stream = subscriptions.find(x => x.name === narrow[0].operand) || {
-      name: '',
-      invite_only: false,
-      in_home_view: false
-    };
+    const { narrow, subscriptions, streams, color } = this.props;
+    const stream = subscriptions.find(x => x.name === narrow[0].operand) ||
+      { ...streams.find(x => x.name === narrow[0].operand), in_home_view: true } ||
```

Comment:
Do you need to search both subscriptions and streams?
Streams contain all items already.

I am also suspicious of the use of `in_home_view` in this way...